saleor/graphql/account/filters.py

```diff
@@ -51,6 +51,7 @@ def filter_search(qs, _, value):
 
 
 class CustomerFilter(MetadataFilterBase):
+    ids = GlobalIDMultipleChoiceFilter(field_name="id")
```

Comment:
Let's use a `help_text` to add info about Saleor version where this filter was added (`ADDED_IN_38`)
```
    stock_availability = EnumFilter(
        input_class=StockAvailability,
        method="filter_stock_availability",
        help_text="Filter by variants having specific stock status.",
    )
```